data/sql/updates/pending_db_auth/redo-auth.sql

```diff
@@ -0,0 +1,117 @@
+ALTER TABLE `account`
```

Comment:
PR blocked until there is an equivalent PR for all the main tools depending on all the tables changed here. And it's a lot of work.